### ISSUE TITLE
Can openrouter api be supported?

### ISSUE THREAD

**Issue by hekmon8** [Bin-Huang/chatbox#1246]
It seems something went wrong… version1.2.2

![image](https://github.com/Bin-Huang/chatbox/assets/35729939/34ddc443-16c6-43f9-928a-f9ca14ef1822)

**Comment by kaptainkangaroo** [comment]
I got the same error. Please add openrouter.ai support

**Comment by hekmon8** [comment]
你好，我已经收到，谢谢！

**Comment by hekmon8** [comment]
> I got the same error. Please add openrouter.ai support

Is there any way to solve this issue?

**Comment by rodion-m** [comment]
The same error. Very sad, guys. @Bin-Huang

**Comment by Podolyakofs** [comment]
Another question here - is there any way to enable reasoning part from openrouter api - https://openrouter.ai/docs/parameters#include-reasoning ?

**Comment by Xi-Gong** [comment]
> Another question here - is there any way to enable reasoning part from openrouter api - https://openrouter.ai/docs/parameters#include-reasoning ?

Yes, hope can support soon with reasoning content. in the docs add `include_reasoning: true` can achieve this.
